### PR TITLE
Add 'package' script to build a webapp bundle

### PR DESCRIPTION
**Pull request by dbkr**
for straight unzipping on a production server.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "build:compile": "babel --source-maps -d lib src",
     "build:bundle": "NODE_ENV=production webpack -p lib/vector/index.js vector/bundle.js",
     "build": "npm run build:css && npm run build:compile && npm run build:bundle",
+    "package": "npm run build && mkdir packages && bestzip packages/vector-`git describe --dirty || echo unknown`.zip vector",
```
Comment (richvdh) [inline package.json]: you want `mkdir -p`.

Also, I'm led to wonder why we bother with `bestzip`, given we're assuming other linuxy things like `mkdir` and `git` and `&&`, but whatever.

**Comment by richvdh** [comment]
lgtm otherwise

File: package.json (hunk 1 above)
Comment (dbkr) [inline package.json]: mkdir is on Windows, mkdir -p is not - hence plain mkdir, although I realise of course && won't work so I've replaced it with ';'. We don't rely on git - we just use 'unknown' if the git command doesn't work (nonetheless it is very tempting to drop windows support for this given it's so annoying, but this just about seems doable).

**Comment by richvdh** [comment]
Ok, but now it will appear to make a zip when the buildstep fails?

**Comment by dbkr** [comment]
Good point. This should be better.

**Comment by richvdh** [comment]
lgtm

**Pull request merged**